src/util/projects.tsx

```diff
@@ -8,13 +8,7 @@ export const projectSubpages = [
   "configuration",
 ];
 
-export const getProjectFromUrl = (url: string) => {
-  const parts = url.split("/");
-  if (projectSubpages.includes(parts[3])) {
-    return parts[2];
-  }
-  return undefined;
-};
+export const getProjectFromUrl = (url: string) => url.split("/")[2];
```

Comment:
This breaks, because we also have pages like https://lxd-ui-314.demos.haus/ui/settings and currently this function returns "settings" as project. We should keep this logic as is.

Better add a redirect via custom component to the new route.